snippets/csharp/VS_Snippets_CLR_System/system.GenericIEquatable.Equals/cs/EqualsEx2.cs

```diff
@@ -11,15 +11,14 @@ public class Person : IEquatable<Person>
    public Person(string lastName, string ssn)
    {
       if (Regex.IsMatch(ssn, @"\d{9}"))
-         uniqueSsn = String.Format("{0}-(1}-{2}", ssn.Substring(0, 3),
+         uniqueSsn = String.Format("{0}-{1}-{2}", ssn.Substring(0, 3),
```

Comment:
We've been pushing string interpolation whenever we see these statements.  Can you change this to:

```
uniqueSsn = $"{ssn.Substring(0, 3)}--{ssn.Substring(3, 2)}--{ssn.SubString(5, 4)};
```